src/IniFileParser/Parser/IniDataParser.cs

```diff
@@ -350,7 +350,7 @@ protected virtual void ProcessKeyValuePair(string line, IniData currentIniData)
             // get key and value data
             string key = ExtractKey(line);
 
-			if (string.IsNullOrEmpty(key) && Configuration.SkipInvalidLines) return;
+            if (string.IsNullOrEmpty(key) && Configuration.SkipInvalidLines) return;
```

Comment:
That change should be done in a separate commit. this PR isn't intended to change behavior.